cube_tables.cpp

```diff
@@ -199,4 +199,12 @@ const Vector3i g_ordered_moore_area_3d[MOORE_AREA_3D_COUNT] = {
 	Vector3i(1, 1, 1)
 };
 
+const float g_side_tangents[SIDE_COUNT][4] = {
+	{0.0f, 0.0f, -1.0f, -1.0f},
+	{0.0f, 0.0f, 1.0f, -1.0f},
+	{1.0f, 0.0f, 0.0f, -1.0f},
+	{-1.0f, 0.0f, 0.0f, -1.0f},
+	{1.0f, 0.0f, 0.0f, -1.0f},
```

Comment:
This array has only 5 elements, but cubes have 6 sides. How curious this compiled for you Oo

Reply:
Uhhhh. Lmao you're not wrong. Apparently I'm writing tangents to 4 of the sides? So it only having 5 isn't breaking it. It's very curious this makes it through the visual server without dying.

I'll fix it and the loop. Starting to feel like I'm losing brain cells with my recent programming.

Reply:
No I think your code is correct, it is really adding to the 6 faces because the outer loop iterates from 0 to `SIDE_COUNT`. The next loop runs 4 times for the 4 vertices, and the innermost loop runs 4 times because each tangent has 4 elements.